docs/dev/swagger-source.json

```diff
@@ -1748,7 +1748,14 @@
 						"in": "query",
 						"required": false,
 						"type": "string"
-					}
+					},
+                    {
+                        "name": "stageIds",
+                        "description": "Filter results by email templates that belong to one or more workflow stage IDs. Separate role IDs with a comma.",
+                        "in": "query",
+                        "required": false,
+                        "type": "string"
+                    }
```

Comment:
Thanks for thinking of the documentation! Looks like this needs a spaces-to-tabs change but otherwise good. Maybe add info about the other category:

> Filter results by email templates that belong to one or more workflow stage IDs. Separate role IDs with a comma. Pass `0` to return templates not assigned to a stage.